src/charts/donut/donut-pct-chart.html

```diff
@@ -1,5 +1,22 @@
-<span>
-  <pf-c3-chart ng-if="$ctrl.data.dataAvailable !== false" id="{{$ctrl.donutChartId}}" config="$ctrl.config"
-               get-chart-callback="$ctrl.setChart"></pf-c3-chart>
-  <pf-empty-chart ng-if="$ctrl.data.dataAvailable === false" chart-height="$ctrl.chartHeight"></pf-empty-chart>
+<span class="pct-donut-chart-pf">
+  <span ng-class="{'pct-donut-chart-pf-block': $ctrl.config.label.orientation !== 'left' && $ctrl.config.label.orientation !== 'right', 'pct-donut-chart-pf-left': $ctrl.config.label.orientation === 'left', 'pct-donut-chart-pf-right': $ctrl.config.label.orientation === 'right'}">
+    <span class="pct-donut-chart-pf-chart">
+      <pf-c3-chart ng-if="$ctrl.data.dataAvailable !== false" id="{{$ctrl.donutChartId}}" config="$ctrl.config"
+                   get-chart-callback="$ctrl.setChart"></pf-c3-chart>
+      <pf-empty-chart ng-if="$ctrl.data.dataAvailable === false" chart-height="$ctrl.chartHeight"></pf-empty-chart>
+    </span>
+    <span ng-if="$ctrl.data.dataAvailable !== false && $ctrl.config.label && !$ctrl.config.label.labelFn()" class="pct-donut-chart-pf-label">
+      {{$ctrl.config.label.title}}
+      <span ng-if="$ctrl.data" ng-switch="$ctrl.config.label.label">
```

Comment:
```$ctrl.config.label.label``` I think we do need 'labelConfig' or something clearer.  Currently, 'label.label' is tough to read.